project/Build.scala

```diff
@@ -12,7 +12,7 @@ object DottyBuild extends Build {
   val baseVersion = "0.1"
   val isNightly = sys.env.get("NIGHTLYBUILD") == Some("yes")
 
-  val jenkinsMemLimit = List("-Xmx1300m")
+  val jenkinsMemLimit = List("-Xmx1500m")
```

Comment:
The bugfix is at: https://github.com/lampepfl/dotty/pull/1636/commits/7f4f387f220478ce4f8f13b99c7d16f0c254339d#diff-ca99a4570465b856d9f97804352f6851R167
compare with the implementation of sources in https://github.com/scala/scala-partest/blob/7c265b38352f146f941a873b153cbfc22d7b7afd/src/main/scala/scala/tools/partest/nest/Runner.scala#L427